metricbeat/module/rabbitmq/_meta/config.yml

```diff
@@ -1,2 +1,10 @@
 - module: rabbitmq
+  #metricsets:
+  #  - node
+  #  - queue
+  #  - connection
+  period: 10s
   hosts: ["localhost:15672"]
+
+  username: guest
```

Comment:
Should this be commented out or is `guest` a common default?

Reply:
Yes, guest exists by default.

Reply:
So it's better to connect by default with `guest/guest` the have no credentials?

Reply:
`guest`/`guest` can be seen as some kind of "no credentials" for rabbitmq :)
By default it always exists, and I don't know if it is possible to connect with rabbitmq without credentials. I think it'd be good if metricbeat tries to connect with these credentials by default too, so it can collect some metrics out of the box.

Reply:
SGTM. Not part of this PR but I wonder if we should in these case then also set them directly in the code as defaults.

Reply:
Yes, good point, I'll take care of this.